gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerFragment.java

```diff
@@ -152,6 +153,9 @@ private void onMapReady(MapAdapter map) {
     homeScreenViewModel
         .getBottomSheetState()
         .observe(this, state -> onBottomSheetStateChange(state, map));
+    binding.addPolygons.addPolygonButton.setOnClickListener(__ -> addPolygonVertices(map));
```

Comment:
Please call these using event bindings in layout instead of setting listeners programmatically.